examples/navier-stokes/navierstokes.c

```diff
@@ -895,7 +895,6 @@ int main(int argc, char **argv) {
   CHKERRQ(ierr);
 
   // Clean up
-  CeedVectorDestroy(&xceed);
```

Comment:
Given my druthers, I'd rather destroy xceed and xcorners here after use like we do with q0ceed, multevec, and multlvec.

Reply:
Ok. Put it back there and also moved the other ones for onesvec and xcorners for consistency